### PR TITLE
I#67 add-friends-to-group

### PR DESCRIPTION
**Pull request by ryanknauer**
#67

### CODE REVIEW

File: api-customer/functions/AddFriendsToGroup/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        const joinString = "," + pending_game_group_id.toString() + "),(";
+        const values = "(" +  group.join(joinString) + "," + pending_game_group_id.toString() + ");";
+
+        var client = yield pool.connect();
+        try {
+            var query = SQL `
+                INSERT INTO GAME_GROUP_INVITE (invited, pending_game_group)
+                VALUES
+                `;          
+            query.append(values);
```
Comment (anmolg) [inline api-customer/functions/AddFriendsToGroup/index.js]: Any issues with using the typical approach?

File: api-customer/functions/AddFriendsToGroup/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            yield client.query('BEGIN');
+            let result = yield client.query(query);
```
Comment (anmolg) [inline api-customer/functions/AddFriendsToGroup/index.js]: if we aren't doing anything with result, then we can ignore the let field

File: api-customer/functions/AddFriendsToGroup/index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            yield client.query('COMMIT');
+
+            let response = {
+                statusCode: 200
+            };
+            callback(null, response);
```
Comment (anmolg) [inline api-customer/functions/AddFriendsToGroup/index.js]: console.log the response

File: api-customer/functions/AddFriendsToGroup/index.js (hunk 1 above)
Comment (ryanknauer) [inline api-customer/functions/AddFriendsToGroup/index.js]: Seems all good to me. Forsee any problems?

Comment (shreyansjain) [inline api-customer/functions/AddFriendsToGroup/index.js]: values = `'(35,1),(36,1),(37,1);'`

**Pull request merged**